src/app/vault/add-edit-custom-fields.component.html

```diff
@@ -49,7 +51,16 @@ <h3 class="mt-4">{{'customFields' | i18n}}</h3>
                             </button>
                         </div>
                     </div>
+                    <!-- Linked -->
+                    <div class="input-group" *ngIf="f.type === fieldType.Linked">
+                        <select id="fieldValue{{i}}" name="Field.Value{{i}}" class="form-control" [(ngModel)]="f.linkedId"
+                            *ngIf="f.type === fieldType.Linked && cipher.linkedFieldOptions != null"
+                            [disabled]="cipher.isDeleted || viewOnly">
+                            <option *ngFor="let o of linkedFieldOptions" [ngValue]="o.value">{{o.name}}</option>
+                        </select>
+                    </div>
```

Comment:
I suggest adding a disabled input or a span with `type=password` and our typical eye to show the value, which would be filled in with the linked value.

For each of these custom field types, the `value` box is the indicator of the custom field type, it should be here too.

Reply:
n/m this is a bad idea from me not thinking through the layout. sorry